<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GainSetupDialog</class>
 <widget class="QDialog" name="GainSetupDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>280</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GainSetupDialog</string>
  </property>
  <widget class="QPushButton" name="pbOk">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>230</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbCancel">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>230</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbHgb">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>120</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>HGB</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbWbc">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>20</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>WBC</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbRbc">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>70</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>RBC</string>
   </property>
  </widget>
  <widget class="DigitLineEdit" name="leWbc">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>20</y>
     <width>200</width>
     <height>35</height>
    </rect>
   </property>
  </widget>
  <widget class="DigitLineEdit" name="leRbc">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>70</y>
     <width>200</width>
     <height>35</height>
    </rect>
   </property>
  </widget>
  <widget class="DigitLineEdit" name="leHgb">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>120</y>
     <width>200</width>
     <height>35</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbHgbAd">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>170</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>HGB AD</string>
   </property>
  </widget>
  <widget class="DigitLineEdit" name="leHgbAd">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>170</y>
     <width>200</width>
     <height>35</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>LineEdit</class>
   <extends>QLineEdit</extends>
   <header>lineedit.h</header>
  </customwidget>
  <customwidget>
   <class>DigitLineEdit</class>
   <extends>LineEdit</extends>
   <header>digitlineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
